src/backend/distributed/deparser/deparse_database_stmts.c

```diff
@@ -178,3 +181,27 @@ DeparseAlterDatabaseRefreshCollStmt(Node *node)
 
 
 #endif
+
+static void
+AppendAlterDatabaseSetStmt(StringInfo buf, AlterDatabaseSetStmt *stmt)
+{
+	appendStringInfo(buf, "ALTER DATABASE %s ", quote_identifier(stmt->dbname));
```

Comment:
This is causing a doubled space in the output.
```suggestion
	appendStringInfo(buf, "ALTER DATABASE %s", quote_identifier(stmt->dbname));
```